source/cloud/cloud-administration/sso-google.md

```diff
@@ -2,49 +2,77 @@
+
+3. Enter "Mattermost - your-company-name" as the **Project Name**, then select **Create**.
 
 ![](../../../source/images/google_1_credentials.png)
 
-2. In the **Dashboard** click the option to **ENABLE APIS AND SERVICES**.
+### Step 2: Enable APIs and Services
 
-![](../../../source/images/google_enable_api.png)
+1. In the **Dashboard** select the option to **ENABLE APIS AND SERVICES**.
 
-Once in the API Library, scroll down to [Google People API](https://console.developers.google.com/apis/api/plus/overview) and click **Enable** in the header. This might take a few minutes to propagate through Google's systems.
+2. In the API Library, scroll down to [Google People API](https://console.developers.google.com/apis/api/plus/overview), then select **Enable** in the header. This might take a few minutes to propagate through Google's systems.
 
-3. Leave the Google+ API menu to return to the **APIs & Services menu**. In the left hand sidebar select **Credentials**, click the **OAuth consent screen** header and enter "Mattermost" as the **Application Name**, then click **Save**.
+![](../../../source/images/google_enable_api.png)
 
-![](../../../source/images/google_3_oauth_consent_screen.png)
+### Step 3: Add Mattermost as an authorized domain
 
-Scroll further down to add your Mattermost instance to a list of authorized domains that can access the API. For example, if it is hosted on `mattermost.yourdomain.com` add `yourdomain.com`.
+1. Leave the **Google+ API menu** to return to the **APIs & Services menu**. 
 
-![](../../../source/images/google_authorised_domains.png)
+2. In the left-hand sidebar, select **Credentials**, select the **OAuth consent screen** header, enter "Mattermost" as the **Application Name**, then select **Save**.
 
-4. Select **Credentials** and click **Create credentials**, then choose **OAuth client ID** from the drop-down list.
+3. Scroll further down to add your Mattermost instance to a list of authorized domains that can access the API. For example, if it is hosted on `mattermost.yourdomain.com` add `yourdomain.com`.
 
-![](../../../source/images/google_3_oauth_client_id.png)
+4. Select **Credentials** and select **Create credentials**, then choose **OAuth client ID** from the drop-down list.
 
 5. Select **Web Application** as the **Application type**, and choose a descriptive **Name** for the OAuth connection.
 
-Under **Restrictions > Authorized redirect URIs**, enter `your-mattermost-url/signup/google/complete` (example: `http://localhost:8065/signup/google/complete`). Click **Create**.
+6. Under **Restrictions > Authorized redirect URIs**, enter `your-mattermost-url/signup/google/complete` (example: `http://localhost:8065/signup/google/complete`), then select **Create**.
 
-![](../../../source/images/google_4_web_app.png)
+7. Copy and paste the **Client ID** and **Client Secret** values to a temporary location. You will enter these values in the Mattermost System Console.
```

Comment:
These steps have changed. Here are the steps as defined in the UI.
![Screen Shot 2021-02-09 at 2 17 35 PM](https://user-images.githubusercontent.com/12704875/107430102-fbd29580-6ae1-11eb-89da-4eda4d705277.png)

I think we will probably need new screen shots as well.